sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReactorAmqpConnection.java

```diff
@@ -159,14 +159,34 @@ public ServiceBusReactorAmqpConnection(String connectionId, ConnectionOptions co
      */
     @Override
     public Mono<AmqpReceiveLink> createReceiveLink(String linkName, String entityPath, ReceiveMode receiveMode,
-        boolean isSession, String transferEntityPath, MessagingEntityType entityType) {
+        String transferEntityPath, MessagingEntityType entityType) {
+        return createReceiveLink(linkName, entityPath, receiveMode, null,
```

Comment:
Since these are performing two different operations, and you have created a separate method in ReactorSession for creating a consumer with a sessionId. This should call that.